### DISCUSSION TITLE
You can't tell which entry you're editing.

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2906] [Feature Requests]
When you're editing an entry, it's very easy to get to a state where you can't tell what entry you're editing. (Basically, are you unable to see the title field? Can't tell what entry you're editing.)

<img width="1482" alt="image" src="https://user-images.githubusercontent.com/308752/147504515-b81c4f8d-4ec4-4864-b21c-9628cc749035.png">

**Describe the solution you'd like**
Add the title to the `.main-nav__title`.

<img width="1477" alt="image" src="https://user-images.githubusercontent.com/308752/147505072-c1c7b924-ec32-4465-9960-4fdce64c0a62.png">

**Comment by intoeetive** [comment]
@ajohnson FYI

**Comment by robsonsobral** [comment]
Actually, we can't tell which member role either.

However, there's already a standard for it. Check channel or site edition. They use the name of the item being edited only.

It would be redundant to have "Edit entry" written twice. Don't you think, @murtaugh ?

**Comment by ajohnson** [comment]
@murtaugh Good point. We'll soon be releasing an update to the Edit Entry screen that includes some visual changes to the header area of the screen that could affect this issue. So let's wait until after those changes are released and we can then determine how to best proceed with this issue. Thanks Tim!

**Comment by intoeetive** [comment]
Just noting that this still hasn't been addressed as of EE 7.2

**Comment by Yulyaswan** [comment]
PR #3864
